fdbserver/workloads/ConfigureDatabase.actor.cpp

```diff
@@ -285,6 +285,13 @@ struct ConfigureDatabaseWorkload : TestWorkload {
 	}
 
 	ACTOR Future<Void> _start(ConfigureDatabaseWorkload* self, Database cx) {
+		// Redwood is the only storage engine type supporting encryption.
```

Comment:
I need to cleanup the same logic in `_setup`. It is moved to `_start` because if ConfigureDatabase workload is run in restart test, the setup phase can be bypassed.